Resources/doc/index.rst

```diff
@@ -83,6 +83,9 @@ application:
         # Whether or not to wrap migrations in a single transaction.
         transactional: true
 
+        # Whether or not to enable the profiler collector. This add some queries overhead.
```

Comment:
```suggestion
        # Whether or not to enable the profiler collector. This adds some queries overhead.
```

Reply:
@greg0ire indeed I can use the same sentence `Use profiler to calculate and visualize migration status.`

do I need to target v3.1? and keep there value `false`

thx

Reply:
The correct branch to target is 3.2, as the last release is 3.2.2